owasp-suppressions.xml

```diff
@@ -80,6 +80,7 @@
    ]]></notes>
     <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
     <cve>CVE-2022-45688</cve>
+    <cve>CVE-2023-5072</cve>
```

Comment:
Now that I look a bit more closely, this one (and its sibling 45688 above it) is also incorrectly suppressed. Both have released. fixes that should be used. They historically had a FP on older versions of the dependency check plugin, but those were resolved last month. Try removing it, upgrading the dep check plugin to latest and rerun.